src/unstract/sdk/file_storage/impl.py

```diff
@@ -201,14 +183,10 @@ def size(self, path: str) -> int:
         Returns:
             int: Size of the file in bytes
         """
-        try:
-            file_info = self.fs.info(path)
-            return file_info["size"]
-        except FileNotFoundError as e:
-            raise e
-        except Exception as e:
-            raise FileOperationError(str(e)) from e
+        file_info = self.fs.info(path)
+        return file_info["size"]
```

Comment:
`.get` might be a better choice here. If `size` is not present in `fileinfo`, using direct access could raise a KeyError

Reply:
that is being caught in the annotation method. So should be fine?  Just wondering how using .get() helps? Won't that give no "attribute" error?

Reply:
Also KeyError/Exception is being handled inside the kip_local_cache annotation...